frontend/public/components/network-policies/_create-network-policy.scss

```diff
@@ -0,0 +1,63 @@
+.co-create-networkpolicy {
+  &__alert {
+    margin-bottom: 0;
+  }
+}
+
+.co-create-networkpolicy__deny .checkbox {
+  padding-right: 15px;
+}
+
+.co-create-networkpolicy__rules-expandable > .pf-c-expandable-section__toggle {
+  font-size: var(--pf-global--FontSize--xl);
+  font-weight: var(--pf-global--FontWeight--bold);
+}
+
+.co-create-networkpolicy__rules-expandable > .pf-c-expandable-section__content {
+  margin-top: -36px;
+}
+
+.co-create-networkpolicy__peer-expandable > .pf-c-expandable-section__content {
+  margin-left: 27px;
+}
+
+.co-create-networkpolicy__expandable-actions {
+  display: flex;
+  flex-direction: row-reverse;
+  margin-bottom: -33px;
+}
+
+.co-create-networkpolicy__peer-type {
+  margin-top: 10px;
+}
+
+.co-create-networkpolicy__header {
+  display: flex;
+}
+
+.co-create-networkpolicy__header label {
+  margin-top: 8px;
+}
+
+.co-create-networkpolicy__header .pf-m-secondary {
+  margin: 8px;
+  top: -8px;
+}
+
+.co-create-networkpolicy__header-right {
+  margin-left: auto;
+}
+
+.co-create-networkpolicy__header .pf-m-link {
+  margin-bottom: 15px;
+}
+
+.co-create-networkpolicy__header .pf-m-secondary {
+  color: var(--pf-global--primary-color--100);
+  padding-left: var(--pf-global--spacer--md);
+  padding-right: var(--pf-global--spacer--md);
+}
+
+.co-create-networkpolicy__header .pf-m-secondary .pf-c-dropdown__toggle-icon {
+  margin-right: 0;
+}
```

Comment:
another user experience issue is several text font size and bold format seems unmatched

<img width="659" alt="Screen Shot 2021-05-20 at 5 13 12 PM" src="https://user-images.githubusercontent.com/12692381/118953043-0cbb5e80-b98f-11eb-9d3e-a45bfc6726c7.png">

Reply:
This comment was already addressed. "Ingress" and "Egress" are similar to Policy type